ElectronClient/.gitignore

```diff
@@ -6,4 +6,5 @@ gui/*.min.js
 plugins/*.min.js
 .DS_Store
 gui/note-viewer/pluginAssets/
-pluginAssets/
\ No newline at end of file
+app/
+pluginAssets/
```

Comment:
Please remove this file from the commit. There's no longer an "app" directory (and if you have one in your workspace, you should remove it).